ui/pages/institutional/custody/custody.js

```diff
@@ -148,7 +151,10 @@ const CustodyPage = () => {
           setSelectedCustodianType(custodian.type);
         } else {
           setMatchedCustodian(custodianByDisplayName);
-          setIsConfirmConnectCustodianModalVisible(true);
+          custodianByDisplayName?.displayName?.toLocaleLowerCase() ===
+          SATURN_DISPLAY_NAME
```

Comment:
Rather than hardcoding custodian display names (also - we should be using neptune instead of saturn) it might make sense to add something to configuration API indicating a custodian supports QR onboarding